lighthouse-core/index.js

```diff
@@ -40,8 +40,8 @@ function lighthouse(url, flags = {}, configJSON) {
     log.setLevel(flags.logLevel);
 
     // Use ConfigParser to generate a valid config file
-    const config = new Config(configJSON, flags.configPath);
-
+    const config = flags.mixedContent ? new Config(require('./config/mixed-content.js'), null)
```

Comment:
how about we handle this over in bin.js where we handle `--perf` :)

https://github.com/GoogleChrome/lighthouse/blob/3030b4f0651fc4eb2634600d7cf0dbca43972b32/lighthouse-cli/bin.js#L57-L59

Reply:
Good idea. That's a better place than the awkward ternary I came up with.